model/notification/notification.go

```diff
@@ -290,16 +282,14 @@ func (n *Notification) SetTaskMetadata(ID string, execution int) {
 }
 
 type NotificationStats struct {
-	GithubPullRequest  int `json:"github_pull_request" bson:"github_pull_request" yaml:"github_pull_request"`
-	JIRAIssue          int `json:"jira_issue" bson:"jira_issue" yaml:"jira_issue"`
-	JIRAComment        int `json:"jira_comment" bson:"jira_comment" yaml:"jira_comment"`
-	EvergreenWebhook   int `json:"evergreen_webhook" bson:"evergreen_webhook" yaml:"evergreen_webhook"`
-	Email              int `json:"email" bson:"email" yaml:"email"`
-	Slack              int `json:"slack" bson:"slack" yaml:"slack"`
-	GithubMerge        int `json:"github_merge" bson:"github_merge" yaml:"github_merge"`
```

Comment:
You removed the GithubMerge Notification stat but didn't remove it from the subscriber types; is this intentional?

Reply:
there's a todo to remove it at the end of the project. The comment actually wasn't worded such that I could find it, but I changed it